pom.xml

```diff
@@ -100,10 +100,11 @@ The MIT License (MIT)
       <plugins>
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.7.0</version>
+          <version>3.8.0</version>
           <configuration>
             <source>1.8</source>
             <target>1.8</target>
+            <useIncrementalCompilation>false</useIncrementalCompilation>
```

Comment:
Reading this answer the default of true should always be used.

https://stackoverflow.com/questions/16963012/maven-compiler-recompile-all-files-instead-modified

Reply:
@svendiedrichsen wanna open an issue for this?

Reply:
As long as nobody is experiencing any problems. No.